tests/.wakeroot

```diff
@@ -1 +1 @@
-{}
+{"log_header":"", "log_header_extra_width":0}
```

Comment:
Ah, if these aren't optional then we need `wake --init` or similar to generate a default `.wakeroot`. We don't want to increase the barrier to entry for a new project

Reply:
I'd much prefer having these be optional for backwards compatibility.  Besides, these feel like something which is more of a user preference than a repo preference, so forcing every repo to include them anyway seems off.

Reply:
They are optional, but the defaults break the tests.

Reply:
Ah, got it.  Overrides for tests is fine!